test/integration/change-streams/change_streams.prose.test.ts

```diff
@@ -992,7 +992,7 @@ describe('Change Stream prose tests', function () {
     });
 
     it('splits the event into multiple fragments', {
-      metadata: { requires: { topology: '!single', mongodb: '>=7.0.0' } },
+      metadata: { requires: { topology: '!single', mongodb: '>=6.0.9' } },
```

Comment:
The prose tests specify the following:

```
   #. Run only against servers ``>=6.0.9 && <6.1`` or ``>=7.0``.
```

Do we also need to restrict the 6.x runs to <6.1 as well?

Reply:
There is no 6.1, and if there ever is (there won't be) it will have this feature (and if for a wacky reason it doesn't it'd be good to know 😅)